onnx/common/model_helpers.cc

```diff
@@ -7,7 +7,6 @@
 #include "onnx/string_utils.h"
 
 namespace ONNX_NAMESPACE {
-using namespace Common;
```

Comment:
it's ok to have "using namespace in .cc", so you may revert changes in this file.

Reply:
It caused trouble in graph.cc.. I can sync with u offline for this matter